<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Line_Widget</class>
 <widget class="QWidget" name="Line_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainVerticalLayout">
   <item>
    <widget class="QLabel" name="label_main_title">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Line Data Management</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="topControlsLayout">
     <item>
      <layout class="QVBoxLayout" name="leftVerticalLayout">
       <item>
        <layout class="QHBoxLayout" name="exportTypeLayout">
         <item>
          <widget class="QLabel" name="label_export_type">
           <property name="text">
            <string>Export Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="export_type_combo">
           <item>
            <property name="text">
             <string>CSV</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Binary</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="stacked_saver_options">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="CSVLineSaver_Widget" name="csv_line_saver_widget"/>
         <widget class="BinaryLineSaver_Widget" name="binary_line_saver_widget"/>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="export_media_frames_checkbox">
         <property name="text">
          <string>Export matching media frames</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MediaExport_Widget" name="media_export_options_widget" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_left">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionButtonsLayout">
     <item>
      <widget class="QLabel" name="moveToLabel">
       <property name="text">
        <string>Move To:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="moveToComboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="moveLineButton">
       <property name="text">
        <string>Move Line</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteLineButton">
       <property name="text">
        <string>Delete Line</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="actionButtonSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSVLineSaver_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/Lines/CSV/CSVLineSaver_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BinaryLineSaver_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/Lines/Binary/BinaryLineSaver_Widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MediaExport_Widget</class>
   <extends>QWidget</extends>
   <header>IO_Widgets/Media/MediaExport_Widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
